backend/kernelCI_app/management/commands/data/tree-proof.yml

```diff
@@ -0,0 +1,157 @@
+trees:
```

Comment:
This looks good. We can use this as starting point. A few more things:

* rename it to `trees-name.yaml`
* move the `data` dir to `backend/`